src/customers/login-as-customer.md

```diff
@@ -21,6 +21,21 @@ _Login As Customer_
 
 Enabling Login as Customer requires that you enable the feature in your Magento instance and then enable access for Magento Admin users in the user role permissions.
 
+## Customer permission for remote shopping assistance
+
+The Magento Admin can login as a customer, only after the customer allows remote access.
```

Comment:
Thank you for your contributions, @ZhovnirMaria 

I would like to simplify a bit your sentence and add a preliminary explanations to steps below. Something like that:

```
To allow the ability to login as a customer for Magento admin user, a customer should make the next steps:
```

Thanks

Reply:
Hi, @atwixfirster
I made some changes to the request, you can check them.
Thank you!

Reply:
Thank you, @ZhovnirMaria !

@dobooth , could you please review if the suggested changes are correct here?

Thank you!

CC: @leslietilling